sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -1097,14 +1097,16 @@ void setLogSerializer(LogSerializer logSerializer) {
      * @param thread    thread where crash occurred.
      * @param throwable uncaught exception or error.
      */
-    void saveUncaughtException(Thread thread, Throwable throwable) {
+    public UUID saveUncaughtException(Thread thread, Throwable throwable) {
```

Comment:
We do not have public methods at instance level. All public APIs are meant to be accessed from static and be documented.

Reply:
I see. Then perhaps I could introduce a public static method, (let's call it saveExceptionReturnReportUUID) that will call UUID saveUncaughtException(Thread thread, Throwable throwable) internally and return the UUID. Then I'll call saveExceptionReturnReportUUID in my custom exception handler.

Will that be fine?

Reply:
After a discussion with the team, we decided not to introduce a new static API for this. However, we agreed to merge this PR with the explicit note that this method can be changed in future versions without warning.